packages/web/src/common/store/cache/tracks/sagas.ts

```diff
@@ -245,18 +245,19 @@ function* watchEditTrack() {
 function* deleteTrackAsync(
   action: ReturnType<typeof trackActions.deleteTrack>
 ) {
-  const audiusBackendInstance = yield* getContext('audiusBackendInstance')
   yield* waitForWrite()
-  const userId = yield* select(getUserId)
-  if (!userId) {
+  const user = yield* select(getAccountUser)
+  if (!user) {
     yield* put(signOnActions.openSignOn(false))
     return
   }
-  const handle = yield* select(getUserHandle)
+  const userId = user.user_id
+
+  const track = yield* select(getTrack, { id: action.trackId })
+  if (!track) return
 
   // Before deleting, check if the track is set as the artist pick & delete if so
-  const socials = yield* call(audiusBackendInstance.getSocialHandles, handle!)
-  if (socials.pinnedTrackId === action.trackId) {
```

Comment:
Whoa what? We were storing pinned tracks with social info?

Reply:
we used to, but we did move this already like a year.5 ago